GitUI/CommandsDialogs/AboutBoxDialog/FormContributors.cs

```diff
@@ -34,12 +34,11 @@ void InitialiseComponent()
                     tabPages[i] = GetNewTabPage(textBoxes[i], tabCaptions[i]);
                 }
 
-                const string NEWLINES = @"\r\n?|\n";
                 textBoxes[0].Text = string.Format("{0}:\r\n{1}\r\n\r\n{2}:\r\n{3}",
-                    _team.Text, Regex.Replace(Resources.Team, NEWLINES, " "),
-                    _contributors.Text, Regex.Replace(Resources.Coders, NEWLINES, " "));
-                textBoxes[1].Text = Regex.Replace(Resources.Translators, NEWLINES, " ");
-                textBoxes[2].Text = Regex.Replace(Resources.Designers, NEWLINES, " ");
+                    _team.Text, NewlineRegex().Replace(Resources.Team, " "),
+                    _contributors.Text, NewlineRegex().Replace(Resources.Coders, " "));
+                textBoxes[1].Text = NewlineRegex().Replace(Resources.Translators, " ");
+                textBoxes[2].Text = NewlineRegex().Replace(Resources.Designers, " ");
```

Comment:
Can we use a variable instead of multiple invocations?

Reply:
Optimizing one function call does not make any difference, just one more line of code.

```
        [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Text.RegularExpressions.Generator", "8.0.9.3103")]
        private static partial global::System.Text.RegularExpressions.Regex NewlineRegex() => global::System.Text.RegularExpressions.Generated.NewlineRegex_1.Instance;

...
    [GeneratedCodeAttribute("System.Text.RegularExpressions.Generator", "8.0.9.3103")]
    [SkipLocalsInit]
    file sealed class NewlineRegex_1 : Regex
    {
        /// <summary>Cached, thread-safe singleton instance.</summary>
        internal static readonly NewlineRegex_1 Instance = new();
```